source/getting-started/iox2-debugging.md

```diff
@@ -0,0 +1,399 @@
+# Command Line Tools And Debugging
```

Comment:
@elfenpiff Could you rename this file `command-line-tools-and-debugging.md` - the name appears in the URL so it's good to be descriptive.